pkg/apis/serving/v1alpha2/configmap.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha2
```

Comment:
Not that we have to follow it, but I quite like knative's pattern here: https://github.com/knative/serving/blob/master/pkg/apis/config/defaults.go

I see types as decoupled from the configmap. The controller may serve multiple APIs, but can only read one configmap.

Reply:
I looked through, they actually use context to pass along the default values from configmaps.
https://github.com/knative/serving/blob/master/pkg/apis/serving/v1/service_defaults.go#L31